editor/src/components/canvas/ui/floating-insert-menu.tsx

```diff
@@ -91,63 +92,147 @@ function useGetInsertableComponents(filterString: string): InsertableComponentFl
           dependencies,
           fullPath,
         ),
-      ).filter((insertableComponent) =>
-        insertableComponent.name.toLowerCase().includes(filterString.toLowerCase()),
       )
     }
-  }, [packageStatus, propertyControlsInfo, projectContents, dependencies, fullPath, filterString])
+  }, [packageStatus, propertyControlsInfo, projectContents, dependencies, fullPath])
 
   return insertableComponents
 }
 
-export const ListItem: React.FunctionComponent<{
-  highlighted: boolean
-  insertableComponent: InsertMenuItem
-  onClick: (insertableComponent: InsertMenuItem) => void
-  onMouseOver: (insertableComponent: InsertMenuItem) => void
-}> = betterReactMemo(
-  'ListItem',
-  ({ onClick, onMouseOver, insertableComponent, highlighted, ...props }) => {
-    const colorTheme = useColorTheme()
-    const handleClick = React.useCallback(() => {
-      onClick(insertableComponent)
-    }, [insertableComponent, onClick])
-
-    const handleMouseOver = React.useCallback(() => {
-      onMouseOver(insertableComponent)
-    }, [insertableComponent, onMouseOver])
+function useComponentSelectorStyles(): StylesConfig {
+  const colorTheme = useColorTheme()
+  // componentSelectorStyles will only be recreated if the theme changes, otherwise we re-use the same object
+  return React.useMemo(
+    () => ({
+      container: (styles) => ({
+        // the outermost element. It contains the popup menu,  so don't set a height on it!
+        // shouldn't contain any sizing
+        // ...styles,
```

Comment:
There are a lot of `// ...styles`. Can you delete those please.

Reply:
Discussed elsewhere. We've agreed to disagree on this, with the outcome being that this should be left to highlight that we are explicitly omitting the passed in `styles`.